uui-components/src/layout/flexItems/FlexSpacer.tsx

```diff
@@ -4,4 +4,4 @@ import { IAdaptiveItem } from '@epam/uui';
 
 export interface FlexSpacerProps extends IAdaptiveItem {}
 
-export const FlexSpacer = (props: FlexSpacerProps) => <div { ...props } className={ css.flexSpacer } />;
\ No newline at end of file
+export const FlexSpacer = (props: FlexSpacerProps) => <hr role="separator" { ...props } className={ css.flexSpacer } />;
```

Comment:
This is not separator, it flex element which helps to fill space between flex items. Often used to position elements at the beginning and end of a line with space between them.